examples/simple_shield/Prover.toml

```diff
@@ -2,6 +2,6 @@ value = 0
 priv_key = 0
 note_root = 0
 index = 0
-note_hash_path = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+# note_hash_path = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
Sorry I forgot to remove this un-needed comment from my PR, can we remove it here?